codecov.yml

```diff
@@ -1,17 +1,35 @@
 codecov:
   require_ci_to_pass: yes
+  notify:
+    wait_for_ci: no
 
-# coverage of each PR must be at least 85% and must not be negative
+# Coverage of backend PRs must be at least 80% with 5% leeway.
 coverage:
   status:
     project:
       default:
         informational: true
+      backend:
+        target: 37%
+        threshold: 3%
+        flags: 
+          - backend
```

Comment:
The comment indicates that coverage must be 80% +- 5%.  So it's not super clear to me why the "project" section is half of that, while the "patch" section matches the comment.

The caveat here is that I'm not an expert on Codecov, so if this would be intuitive to someone who is likely to manage the file, great.  But I wanted to call it out in case it might also be confusing to people who _aren't_ expected to be confused. :sweat_smile:

Reply:
Altered the comment to clarify the difference!